storage/remote/write_test.go

```diff
@@ -19,9 +19,12 @@ import (
 	"testing"
 	"time"
 
+	config_util "github.com/prometheus/common/config"
 	"github.com/prometheus/common/model"
+
```

Comment:
unintended newline?

Reply:
no the convention for package imports are:

```
stdlib packages

3rd party packages

local packages
```

Reply:
They're 3rd party because they're in `prometheus/common`?

Reply:
yes, it's a different Go module.